.golangci.yaml

```diff
@@ -6,26 +6,27 @@ run:
 linters:
   enable:
     # defaults
-    - deadcode
     - errcheck
     - gosimple
     - govet
     - ineffassign
     - staticcheck
-    - structcheck
     - typecheck
     - unused
-    - varcheck
     # extra
     - exhaustive
     - exportloopref
     - revive
     - goimports
     - gosec
     - misspell
-    - rowserrcheck
     - errorlint
     - contextcheck
+  disable:
+    - deadcode
+    - rowserrcheck
+    - structcheck
+    - varcheck
```

Comment:
is it already being taken care by other checks and that's why disabled?

Reply:
They're deprecated. I don't think golangci runs them anymore though, so I think this is redundant?

Reply:
It is deprecated: https://github.com/smartcontractkit/chainlink-testing-framework/runs/8079127193?check_suite_focus=true#step:4:33

Reply:
can it be removed then?

Reply:
I will see if removing them from the disabled list does indeed not run them.

Reply:
Removing works and pr is updated